src/helpers/check_cookie.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken');
+
+const checkCookie = (req, cb) => {
+  if (req.cookies.token) {
```

Comment:
sorry should have seen this before: change to `(req.cookies && req.cookies.token)`